contracts/token/ERC20/IERC20.sol

```diff
@@ -20,6 +20,12 @@ interface IERC20 {
   function transferFrom(address from, address to, uint256 value)
     external returns (bool);
 
+  function increaseAllowance(address spender, uint256 addedValue) 
```

Comment:
We shouldn't add these here, since this is just the ERC20 specification, which doesn't include these methods. I'm actually unsure about this, maybe we should have an additional interface that describes these? (e.g. `IERC20Allowance`). Let's leave it like this for now, though.

Reply:
I vote for additional interface named as `ISafeERC20` or `IERC20Allowance`

Reply:
done

Reply:
@Aniket-Engg an even better solution would be to implement the methods inside the library, since that would allow using them with non-OpenZeppelin tokens. Take a look at [this snippet](https://gist.github.com/cwhinfrey/91f63c4f0471308f63d7657f8339062e#file-safeerc20-sol-L49) for how to do so. Thanks @cwhinfrey for the suggestion!

Reply:
@nventuro i did that but we still need a better way set non-zero allowance for an address. I tried something but they were not compatible with standard and consistency so i eliminated them. pushed remaining code.